doc/manpages/Makefile

```diff
@@ -214,3 +214,5 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+%: ;
```

Comment:
doing a make clean results in SPHINXBUILD being called - it should not:
Case 1:
`make[2]: Entering directory `.../doc/manpages'
Makefile:12: *** The 'sphinx-build' command was not found. Make sure you have Sphinx installed, then set the SPHINXBUILD environment variable to point to the full path of the 'sphinx-build' executable. Alternatively you can add the directory with the executable to your PATH. If you don't have Sphinx installed, grab it from http://sphinx-doc.org/.  Stop.
make[2]: Leaving directory `.../doc/manpages'
make[1]: *** [clean-recursive] Error 1
`
Case 2:
`
make[2]: Entering directory `/home/lberger/Code/github/frr/hpd-frr-testing/frr.github/doc/manpages'
sphinx-build2.7 -b man -d _build/doctrees   . _build/man
Making output directory...
Running Sphinx v1.1.3
loading pickled environment... not yet created
building [man]: all manpages
updating environment: 17 added, 0 changed, 0 removed
reading sources... [100%] zebra                                                                   
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... bgpd.8 { } eigrpd.8 { } ospf6d.8 { } ospfd.8 { } isisd.8 { } ospfclient.8 { } ldpd.8 { } nhrpd.8 { } pimd.8 { } mtracebis.8 { } ripd.8 { } ripngd.8 { } zebra.8 { } watchfrr.8 { } vtysh.1 { } frr.1 { } 
build succeeded.

Build finished. The manual pages are in _build/man.
rm -rf _build/*

`